packages/nextjs/app/page.tsx

```diff
@@ -1,17 +1,88 @@
 "use client";
 
+import Link from "next/link";
 import type { NextPage } from "next";
 import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
 import { Address } from "~~/components/scaffold-stark";
 import { useAccount } from "@starknet-react/core";
 import { Address as AddressType } from "@starknet-react/chains";
+import Image from "next/image";
 
 const Home: NextPage = () => {
   const connectedAddress = useAccount();
+
   return (
     <>
       <div className="flex items-center flex-col flex-grow pt-10">
-        <div className="px-5 w-[90%] md:w-[75%]"></div>
+        <div className="px-5">
+          <h1 className="text-center">
+            <span className="block text-2xl mb-2">Welcome to</span>
+            <span className="block text-4xl font-bold">Scaffold-Stark 2</span>
+          </h1>
+          <div className="flex justify-center items-center space-x-2">
+            <p className="my-2 font-medium text-[#00A3FF]">
+              Connected Address:
+            </p>
+            <Address address={connectedAddress.address as AddressType} />
+          </div>
+          <p className="text-center text-lg">
+            Get started by editing{" "}
+            <code className="bg-underline italic text-base font-bold max-w-full break-words break-all inline-block">
+              packages/nextjs/app/page.tsx
+            </code>
+          </p>
+          <p className="text-center text-lg">
+            Edit your smart contract{" "}
+            <code className="bg-underline italic text-base font-bold max-w-full break-words break-all inline-block">
+              YourContract.cairo
```

Comment:
Check this section, we dont suppose to have anything related to 'YourContract' at this stage